.changeset/poor-dolphins-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': patch
+---
+
+Changed the Toggle's explanation to be associated with the switch button as its description instead of being included in the label to provide a better experience to screen reader users.
```

Comment:
This is a bit wordy, I'm open to suggestions on how to phrase this better.

Reply:
I think it's fine. Maybe break it down:

```suggestion
Changed the Toggle's `explanation` to be associated with the switch button as its description instead of being included in its label. This provides a better experience to screen reader users.
```